sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/CSharpControlFlowGraphBuilder.cs

```diff
@@ -836,6 +860,15 @@ private Block BuildBreakStatement(BreakStatementSyntax breakStatement, Block cur
             }
 
             var target = this.BreakTarget.Peek();
+            if (currentBlock is BranchBlock possibleTryBlock &&

```

Comment:
until this, when a `break` happened inside a `try`, the block was short-circuited with an only successor: the **ExitBlock**

this change ensures that, in case of a `break` inside a `try`, the edges to `catch` blocks are kept